templates/terranix/config.nix

```diff
@@ -0,0 +1,2 @@
+{ ... }:
+{}
```

Comment:
@oscar-izval I think it would be nice to add a trivial example here, rather than leaving it empty.